lib/LLVMIRCodeGen/LLVMIRGen.cpp

```diff
@@ -2382,12 +2382,10 @@ void LLVMIRGen::generateLLVMIRForInstr(llvm::IRBuilder<> &builder,
   }
 
   default:
-#ifndef NDEBUG
-    llvm::errs() << "Cannot select the instruction:\n";
-    I->dump(llvm::errs());
-    llvm::errs() << "\n";
-#endif
-    GLOW_UNREACHABLE("ERROR: Cannot select the instruction.");
+    std::string sBuf;
+    llvm::raw_string_ostream s(sBuf);
+    I->dump(s);
+    LOG(DFATAL) << "Cannot select the instruction: " << s.str();
```

Comment:
Correct me if I'm wrong -- this isn't equivalent to the previous code, is it? Before only the printing was wrapped in NDEBUG, but it would always abort even if release. Does this also abort in release?

Reply:
Yes it should be the same. DFATAL == ERROR if NDEBUG and FATAL otherwise

Reply:
But doesn't `GLOW_UNREACHABLE()` abort even in release? This change means in release it instead will log an error but proceed with execution, no?

Reply:
Oh yeah you're right. I misread the original code and thought there was an #else. I'll just LOG(FATAL) then.